admin_manual/maintenance/manually-moving-data-folders.rst

```diff
@@ -0,0 +1,91 @@
+=====================================
+How To Manually Move a Data Directory
+=====================================
+
+If you need to move your ownCloud data directory from its current location to
+somewhere else, here is a manual process that you can take to make it happen.
+
+.. NOTE:: 
+   This example assumes that:
+
+   - The current folder is: `/var/www/owncloud/data`
+   - The new folder is: `/mnt/owncloud`
+   - You're using Apache as your webserver
+
+1. Stop Apache
+2. Use rsync to sync the files from the current folder to the new one the folders -> 
```

Comment:
The end of this sentence does not make sense. Do you mean just?
"from the current folder to the new folder"